### DISCUSSION TITLE
How about PixArt support?

### DISCUSSION THREAD

**Discussion by AnalyticsXXI** [lllyasviel/Fooocus#3162] [General]
Lately people are talking more and more about PixArt Sigma. All reviews I heard are quite positive, most people are saying that it is a potent model indeed, very trainable. And even though right now it lacking photo realism, but with increasing interest and positive reviews, I think it's quite safe to assume that its popularity will grow up, so adding support for it will not be waste of time. Also it has no problems with license or grass.

You can download click-to-run portable model here:
https://civitai.com/models/523149 

![ComfyUI_temp_uyimq_00034_](https://github.com/lllyasviel/Fooocus/assets/166360619/987bedc0-ba17-4ff9-8535-ce00e2badbf0)
![ComfyUI_temp_ydpeg_00051_](https://github.com/lllyasviel/Fooocus/assets/166360619/18231d6e-afd2-4abe-bd41-3607bd8ab898)
![PixArt Sigma Output_00069_](https://github.com/lllyasviel/Fooocus/assets/166360619/0086348d-c0b6-4f67-9a58-3dd0ad9ea531)
![pixelwavesigma240608240608103615_long focal length photography aesthetic RA_00023_](https://github.com/lllyasviel/Fooocus/assets/166360619/c3ced776-ea57-450f-b5e2-d1c9c3eb65ac)

**Comment by IPv6** [comment]
It`s quite good and memory-wise efficient. Although very different to SDXL.
There is also no usual controlnets yet

**Comment by wisdombox44** [comment]
That is a great model that gives you MidJourney-like aesthetics. Its weakest points are hands and full-body images. I am using it in ComfyUI and StableSwarm. I would love to see it in Fooocus too.

**Comment by AnalyticsXXI** [comment]
In the other hand, they are already working on the next model, so Sigma will be outdated soon. But next one should be good enough for adding support in Fooocus. They are trying to meet all community popular requests.
https://www.reddit.com/r/StableDiffusion/comments/1dm91w1/the_next_version_of_pixart/

Also, someone from the community already released 0.9B Sigma finetune:
https://huggingface.co/ptx0/pixart-900m-1024-ft

I think, Pixart is very likely is the next step from SDXL, community loves it and it looks very promising, so instead of working on SD3 support I would recommend to starting moving to Pixart direction. Juggernaut already started.
https://www.reddit.com/r/StableDiffusion/comments/1dm1vh9/juggernaut_could_be_coming_to_pixart_in_the_near/

Also, check those landscapes: https://discord.com/channels/1178685971475214478/1253749852127629337

**Comment by PeterJared** [comment]
Something I see in all these images is a lot of sharpen (darken edges to exaggerate contrast)
https://civitai.com/models/523149

**Reply by AnalyticsXXI** [reply]
> Something I see in all these images is a lot of sharpen 

It's a very early model, current quality is not so great. It is interesting not as much because of its current state, but because it is definitely will be improved by both developers and the community, they already actively working on it. I would say, it's almost sure that it will become a popular model. Not Sigma 0.6B though, they are working on next base model for now, so no hurry to implement it, I just recommend to start looking in the Pixart direction and prepare to implement their models in Fooocus, people will want it soon.

For now, you can easily play with it using this portable tool https://civitai.com/models/523149 (not on Mac though)
